app/services/handover_date_service.rb

```diff
@@ -307,12 +249,28 @@ def expected_time_in_custody_gt_10_months?
       release_date > @offender.sentence_start_date + 10.months
     end
 
+    # We can not calculate the handover date for NPS Indeterminate
+    # with parole cases where the TED is in the past as we need
+    # the parole board decision which currently is not available to us.
     def release_date
       if @offender.indeterminate_sentence?
-        @offender.parole_eligibility_date || @offender.tariff_date
-      else
+        if @offender.tariff_date.present? && @offender.tariff_date > Time.zone.today
```

Comment:
Ditto here – this may just be a personal preference of mine, so feel free to ignore:

```suggestion
        if @offender.tariff_date.present? && @offender.tariff_date.future?
```